app/jinja_filters.py

```diff
@@ -82,7 +85,15 @@ def format_month_year_date(value, date_format='%B %Y'):
 
 @blueprint.app_template_filter()
 def format_datetime(value, date_format='%d %B %Y at %H:%M'):
-    return "<span class='date'>{date}</span>".format(date=datetime.strptime(value, '%Y-%m-%dT%H:%M:%S.%f').strftime(date_format))
+    date = datetime.strptime(value, '%Y-%m-%dT%H:%M:%S.%f')
+    tz_time = pytz.timezone('UTC').localize(date)
+    london_tz = pytz.timezone('Europe/London')
+    london_time = tz_time.astimezone(london_tz)
+    time = london_time.strftime(date_format)
+    if london_time.tzinfo._utcoffset == timedelta(hours=1):
+        return "<span class='date'>{date} GMT (+1)</span>".format(date=datetime.strptime(time, date_format))
```

Comment:
AFAIK technically this would either be UTC+1 or BST.

Reply:
We believed people were more likely to understand GMT compared to UTC.

Reply:
Yep okay - whether a timezone name is understandable or not is very subjective so this seems like a reasonable take on it.